.travis.yml

```diff
@@ -17,21 +17,31 @@ env:
   global:
     # See https://git.io/vdao3 for details.
     - JOBS=1
-  matrix:
+
+jobs:
+  fail_fast: true
+  allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-canary
+
+  include:
+    # runs linting and tests with current locked deps
+
+    - stage: "Tests"
+      name: "Tests"
+      script:
+        - npm run lint:hbs
+        - npm run lint:js
+        - npm test
```

Comment:
Should be `yarn`, right?

Reply:
It'll work either way, but you're right -- this should be `yarn` for consistency. This was just from the default configuration that `ember-cli-update` provided.